raiden/transfer/mediated_transfer/state_change.py

```diff
@@ -237,7 +237,7 @@ def from_dict(cls, data) -> 'ActionInitTarget':
         )
 
 
-class ReceiveLockExpired(StateChange):
+class ReceiveLockExpired(AuthenticatedSenderStateChange):
```

Comment:
These guys should be calling `super().__init__()`, and then you will need to add the authenticated address to the constructors/`__eq__`/`__str__`

Reply:
All done... also updated the class to have a sender constructor arg.